source/tyr/tyr/tasks.py

```diff
@@ -276,3 +276,24 @@ def heartbeat():
             producer = connection.Producer(exchange=exchange)
             producer.publish(task.SerializeToString(), routing_key='{}.task.heartbeat'.format(instance.name))
 
+
+@celery.task()
+def create_autocomplete_depot(name):
+    autocomplete_path = current_app.config['TYR_AUTOCOMPLETE_DIR']
+
+    if os.path.exists(autocomplete_path):
```

Comment:
I don't know if we don't want to create this directory, but at least we should warn if the directory is not correctly created